src/pages/Forever/components/Intro/IntroStories.tsx

```diff
@@ -137,7 +139,7 @@ export const IntroStoriesComponent: React.FC<IntroProps> = ({
     <Page key={0} visible={page === 0}>
       <TextContent>
         {textKeys.FOREVER_INTRO_PAGE_1({
-          REFERRER: stripTrailingCharacter('s', 'Rebecca'),
+          REFERRER: stripTrailingCharacter('s', referrerName),
```

Comment:
oups :D

Reply:
I was reeeeally confused about the behavior until i found this